app/controllers/api/v1/invoice_items/random_invoice_items_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::InvoiceItems::RandomInvoiceItemsController < ApplicationController
+  def show
+    render json: InvoiceItem.order("RANDOM()").first
```

Comment:
@CPowell23 (Iknow this is merged but here's a fun refactor) I like where @sallymacnicholas is going with this. Since we have some repetition, I might even suggest adding a `random` method to `ApplicationRecord` since all of your models inherit from there. Define it once and get it everywhere. Then you'd call `InvoiceItem.random` or `Customer.random`. Big advantage is since this call is database specific we currently would have to update several places in the code if we decided to use MySQL or something else. By containing this in `ApplicationRecord` we could get away with changing in one place.